upgrade/helpers/tasks.py

```diff
@@ -377,19 +460,38 @@ def sync_tools_repos_to_upgrade(client_os, hosts):
     tools_repo = entities.Repository(
         name=toolsrepo_name, product=tools_product, url=tools_repo_url,
         organization=org, content_type='yum').create()
+    logger.info("Entities product {} and repository {} is created successfully".
+                format(tools_product, toolsrepo_name))
+    start_time = job_execution_time("tools repo sync operation")
     entities.Repository(id=tools_repo.id).sync()
+    job_execution_time("tools repo {} sync operation".format(toolsrepo_name), start_time)
+    logger.info("Entities repository sync operation has completed successfully for tool "
+                "repos name {}".format(tools_repo.name))
     cv.repository += [tools_repo]
     cv.update(['repository'])
-    call_entity_method_with_timeout(cv.read().publish, timeout=2500)
+    logger.info("Content view publish operation is started successfully")
+    start_time = job_execution_time("CV_Publish")
+    call_entity_method_with_timeout(cv.read().publish, timeout=5000)
```

Comment:
same here ^^

Reply:
To avoid further any timeout-related failure, we analyzed the log after sync.